### PR TITLE
GOLD-450 Added support to create issues from mutations

### PR DESCRIPTION
**Pull request by josecordaz**

### CODE REVIEW

File: internal/mutation.go [hunk 1]
```diff
@@ -1,13 +1,23 @@
 package internal
 
-import "github.com/pinpt/agent/v4/sdk"
+import (
+	"github.com/pinpt/agent/v4/sdk"
+)
 
 // Mutation is called when a mutation request is received on behalf of the integration
 func (g *GithubIntegration) Mutation(mutation sdk.Mutation) (*sdk.MutationResponse, error) {
 	logger := sdk.LogWith(g.logger, "customer_id", mutation.CustomerID(), "integration_instance_id", mutation.IntegrationInstanceID())
```
Comment (pec1985) [inline internal/mutation.go:9]: with recent updates to agent, we need to remove the global `logger` (ie: `g.logger`) and use the one that comes in the function parameter instead, which already contains the customer id and instance id.
```
logger := mutation.Logger
```

**Review by pec1985 — CHANGES_REQUESTED** [review]

File: internal/mutation.go (hunk 1 above)
Comment (Robindiddams) [reply internal/mutation.go:9]: ditto, and as of v4.0.21 ive included the integration instance id in the logger as well

File: internal/mutation.go [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
+import (
+	"github.com/pinpt/agent/v4/sdk"
+)
 
 // Mutation is called when a mutation request is received on behalf of the integration
 func (g *GithubIntegration) Mutation(mutation sdk.Mutation) (*sdk.MutationResponse, error) {
-	logger := sdk.LogWith(g.logger, "customer_id", mutation.CustomerID(), "integration_instance_id", mutation.IntegrationInstanceID())
-	sdk.LogInfo(logger, "mutation request received", "action", mutation.Action(), "id", mutation.ID(), "customer_id", mutation.CustomerID(), "model", mutation.Model())
+
+	logger := sdk.LogWith(mutation.Logger())
```
Comment (Robindiddams) [inline internal/mutation.go:10]: this is the same as:
```
logger := mutation.Logger()
```

Comment (Robindiddams) [reply internal/mutation.go:10]: I have the global logger removed in #26  if you wanna help me get that merged then you wont have to address this

**Review by Robindiddams — APPROVED** [review]

**Pull request merged**